### PR TITLE
Backported to 3.4 ItKubernetesEvents changes to remove test dependencies

### PR DESCRIPTION
**Pull request by marinakog**
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10327/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10328/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java [hunk 1]
```diff
@@ -749,4 +754,75 @@ public static void shutdownDomainAndVerify(String domainNamespace, String domain
...
+                            + "with ApiException when tried to get domain %s in namespace %s",
+                    domainUid, domainNamespace));
+
+    assertNotNull(domain, "Got null domain resource");
+    assertNotNull(domain.getSpec(), domain + "/spec is null");
+    assertFalse(domainHasRollingCondition(domain), "Found rolling condition at start of test");
+    return domain;
+  }
+
+  private static boolean domainHasRollingCondition(Domain domain) {
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java:823]: I believe these conditions doesn't exist in 3.4. It will not make the tests to fail, so its fine.

**Review by sankarpn — APPROVED** [review]
lgtm

**Comment by xiancao** [comment]
Can you update the pom.xml to replace ItKubernetesEvents to ItKubernetesDomainEvents?

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java [hunk 2]
```diff
@@ -15,8 +15,6 @@
 import java.util.Properties;
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java:15]: update the copyright of ItKubernetesDomainEvents.java
// Copyright (c) 2022, Oracle and/or its affiliates.

Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java:15]: fixed

**Review by anpanigr — COMMENTED** [review]
lgtm

**Review by anpanigr — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java [hunk 3]
```diff
@@ -319,76 +330,47 @@ void testDomainK8sEventsNonExistingCluster() {
...
+      assertNotNull(domain, " Can't create domain resource");
+
+      String originalDomainHome = domain.getSpec().getDomainHome();
+
+      OffsetDateTime timestamp = now();
 
       logger.info("Checking if the admin server {0} is shutdown in namespace {1}",
-          adminServerPodName, domainNamespace1);
-      checkPodDoesNotExist(adminServerPodName, domainUid, domainNamespace1);
+              adminServerPodName, domainNamespace5);
```
Comment (rjeberhard) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java:347]: You have a lot of changed lines where the only change is that you moved the content over another tab stop? Why? If it's just the configuration of your editor, can you update its formatting so that it doesn't make as many unnecessary changes?

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java (hunk 1 above)
Comment (rjeberhard) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java:823]: Let's remove the assertion. It's strange to assert for the lack of a condition that doesn't exist in 3.4.
